phpBB/install/startup.php

```diff
@@ -81,4 +81,5 @@ function phpbb_include_updated($path, $phpbb_root_path, $optional = false)
 
 $phpbb_installer_container = $phpbb_installer_container_builder
 	->with_config_path($config_path)
+	->with_custom_parameters(array('cache.driver.class' => 'phpbb\cache\driver\file'))
```

Comment:
The real issue is that the cache and php file storage are separated but we rely on the cache to purge the stored php files. We need to build a proper API for that but that will not happen before 3.3